test/e2e/serial/tests/configuration.go

```diff
@@ -1510,3 +1523,19 @@ func getLabelRoleWorker() string {
 func getLabelRoleMCPTest() string {
 	return fmt.Sprintf("%s/%s", depnodes.LabelRole, roleMCPTest)
 }
+
+func isCustomPolicySupportEnabled(nro *nropv1.NUMAResourcesOperator) bool {
+	GinkgoHelper()
+
+	const minCustomSupportingVString = "4.18"
```

Comment:
This commit is going to be merge to 4.19 and (probably) backport to 4.18, so why there's a need to check if this code is running against a version older than 4.18

Reply:
because we use and maintain same test image for all versions, so this code will also run on < 4.18 versions, and we do not backport changes done to the serial suite

Reply:
What test image? 
I thought that we are checking out the code according to the cluster version and then run the appropriate tests.

Reply:
It would be problematic to run tests adjusted for 4.19 on 4.17 because there are significant changes between the two. for example we not longer have MachineConfig

Reply:
but if you're checking the code to release-4.17, this piece of code won't be shown there.
What am I missing?

Reply:
@Tal-or this was implemented u/s eventually to and one time d/s as it turned out the best approach to avoid maintenance load as much as possible, check this for the details: 
https://github.com/openshift-kni/numaresources-operator/tree/main/doc/features

Reply:
`minCustomSupportingVString` is hardcoded indeed, but `configuration.PlatVersion` is autodetected from the cluster the suite is running against

Reply:
> but if you're checking the branch to release-4.17, this piece of code won't be shown there.
> What am I missing?

This piece of code will run on all releases and is shipped in our latest quay tests' image. The tl;dr is that the operator controller pod for every release exposes which features are supported per the running version; d/s we scan the serial tests for all the tests that are tagged with the supported feature, that way we run only supported tests. Active features per branch can be found here: 
https://github.com/openshift-kni/numaresources-operator/blob/main/internal/api/features/_topics.json
@Tal-or let me know if you have additional concerns.